include/drivers/clock_control/nrf_clock_control.h

```diff
@@ -15,10 +15,10 @@
 
 /* Define 32KHz clock source */
 #ifdef CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC
-#define CLOCK_CONTROL_NRF_K32SRC 0
+#define CLOCK_CONTROL_NRF_K32SRC IS_ENABLED(CONFIG_SOC_SERIES_NRF91X) ? 1 : 0
```

Comment:
I think it would be better/safer to include `<nrf_clock.h>` and use `NRF_CLOCK_LFCLK_RC` instead.

Reply:
fixed.
Should work, it is then casted again to u32_t and written to `LFCLKSRC`:
https://github.com/zephyrproject-rtos/zephyr/blob/b2a3e12d861a569c3c4cff2168d9165a626bbaee/drivers/clock_control/nrf_power_clock.c#L183